tests/subsys/kmu/pytest/test_kmu_basic.py

```diff
@@ -65,7 +75,11 @@ def test_if_board_does_not_boot_when_image_is_not_signed_with_correct_key(
 
 @pytest.mark.usefixtures("no_reset")
 def test_if_board_does_not_boot_when_any_key_was_provisioned(dut: DeviceAdapter):
```

Comment:
The test name is confusing, implies this tests a scenario where ANY (at least one) key was provisioned, but it actually tests that NO key wa provisioned, I suggest to change it to `test_if_board_does_not_boot_when_no_key_was_provisioned` or `test_if_board_does_not_boot_without_provisioned_keys`